lib/ansible/plugins/cliconf/nxos.py

```diff
@@ -51,6 +52,13 @@ def send_command(self, command, prompt=None, answer=None, sendonly=False, newlin
             resp = self._connection.send_request(command, **kwargs)
         return resp
 
+    def send_config(self, config):
+        if isinstance(self._connection, NetworkCli):
+            resp = self._connection.edit_config(config)
+        else:
+            resp = self._connection.edit_config(config)
```

Comment:
These are the same?

Reply:
@Qalthos nope, these are not the same.
if connection is NetworkCli, edit_config will be called from cliconf, otherwise it will be called from httpapi/nxos.py